README.md

```diff
@@ -34,55 +34,66 @@ This proposal pursues the third option, as we expect it to lead to the best deve
 
 Module attributes have to be made available in several different contexts. This section contains one possible syntax, but there are other options, discussed in [#6](https://github.com/littledan/proposal-module-attributes/issues/6).
 
-Here, a key-value syntax is used, with the key `type` used as an example indicating the module type. Such key-value syntax can be used in various different contexts. Another option is to have just a single string (discussion in [#11](https://github.com/littledan/proposal-module-attributes/issues/11)).
+Here, a single string is permitted to describe a single module attribute, as discussed in [#12](https://github.com/littledan/proposal-module-attributes/issues/12).
+
+Although unspecified in the module attributes proposal, this intention of the proposal champions is that this string would be interpreted as the module type, as shown in the following examples.
 
 The JavaScript standard would basically be responsible for passing the options bag up to the host environment, which could then decide how to interpret it. Issues [#24](https://github.com/littledan/proposal-module-attributes/issues/24) and [#25](https://github.com/littledan/proposal-module-attributes/issues/25) discuss the Web and Node.js feature and semantic requirements respectively, and issue [#10](https://github.com/littledan/proposal-module-attributes/issues/10) discusses how to allow different JavaScript environments to have interoperability.
 
 ### import statements
 
-The ImportDeclaration would allow any arbitrary attributes after the `with` keyword.
-
-For example, in some environments a `type` key could be mandatory to load a module that isn't JavaScript (json, css, html, ...).
```

Comment:
> For example, in some environments a `type` key could be mandatory to load a module that isn't JavaScript (json, css, html, ...).

It might still make sense to have this sentence in here, modified to something like: 
"For example, in some environments the string could be interpreted as indicating module type and be mandatory when loading a module that isn't JavaScript (json, css, html, ...)".

Although perhaps the example speaks for itself.

Reply:
Added an explanation